pages/add-project.tsx

```diff
@@ -0,0 +1,143 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import { useState } from 'react';
+import { useAppSelector } from '../app/hooks';
+import { Navbar } from '../components';
```

Comment:
```suggestion
import { Navbar } from '@/components';
```

Reply:
i think it broke the build

Reply:
Yes btw why we are using @ instead of the ../

Reply:
instead of writing `../../components` we can put `@/components`. it's easy to write and manage

Reply:
But I think it's not working

Reply:
because you don't have the latest main branch code. 
Update your repository and pull the latest code from main branch @AdityaPainuli

Reply:
got it .